lib/matplotlib/pyplot.py

```diff
@@ -4128,6 +4128,8 @@ def streamplot(
     integration_direction="both",
     broken_streamlines=True,
     *,
+    integration_max_step=1.0,
+    integration_max_error=1.0,
```

Comment:
```suggestion
    integration_max_step_scale=1.0,
    integration_max_error_scale=1.0,
```

Can we add `_scale` to the names?  It does make the names long, but without they suggest that they are setting the absolute values of the max_step and max_error which will almost certainly lead to confused (and angry) users.

Reply:
Sure, just added it where necessary